src/scss/includes/direction.scss

```diff
@@ -50,16 +50,18 @@
   overflow: auto;
 }
 
-.itinerary_leg {
-  position: relative;
-
+.itemList-item {
```

Comment:
Make sure to scope that as you are modifying the global `.itemList-item` component here. Or if it's the target look and consistent with other uses in the app, we should modify it in its own SCSS.

Reply:
thanks, this should be a temporary fix, the delimiter has disappeared a few commits ago it seems, and it should be removed in a next commit due to design changes